### ISSUE TITLE
[Feature Request] Add settings button back to home page.

### ISSUE THREAD

**Issue by yodaluca23** [arichorn/uYouPlusExtra#178]
YouTube in one of the recent updates, made the setting cog, button in the top right, only show up in the new "You" tab, if you could add an option to add it back that'd be great, I tried looking through some of the A/B configs, but couldn't find anything mentioning it, if theirs not a built in work around, you may be able to hard code in one, just putting a button their with the same icon, and when clicked have it open the settings page...

**Comment by IncursionX** [comment]
Settings > uyouolus > miscellaneous > “Remove the modern UI” will fix this.

**Comment by yodaluca23** [comment]
> Settings > uyouolus > miscellaneous > “Remove the modern UI” will fix this.

Ok, but I don't want the rest of the Modern UI gone, I only dislike the settings button being gone...

**Comment by IncursionX** [comment]
> > Settings > uyouolus > miscellaneous > “Remove the modern UI” will fix this.
> 
> Ok, but I don't want the rest of the Modern UI gone, I only dislike the settings button being gone...

On my devices, all it does Is change where the settings button is to the old version with your profile on the top right. All other ui elements stay the same

**Comment by yodaluca23** [comment]
> On my devices, all it does Is change where the settings button is to the old version with your profile on the top right. All other ui elements stay the same

Nope, it also removes the new you tab and a bunch of other stuff...
